source/usage-examples/changeStream.txt

```diff
@@ -54,16 +54,18 @@ read events from the change stream below.
    .. tab::
       :tabid: Event
 
-      You can attach listener functions to the ``ChangeStream``  object by calling the ``on()`` method. This method is inherited from the Javascript
-      ``EventEmitter`` class. Pass the string ``"change"`` as the first parameter and your callback function as the
-      second parameter as shown below:
+      You can attach listener functions to the ``ChangeStream``  object
+      by calling the ``on()`` method. This method is inherited from the
+      Javascript ``EventEmitter`` class. Pass the string ``"change"`` as
+      the first parameter and your listener function as the second parameter as shown below:
 
       .. code-block:: javascript
 
-         changeStream.on("change", (changeEvent) => { /* your callback function */ });
+         changeStream.on("change", (changeEvent) => { /* your listener function */ });
 
-      The callback function triggers when a change event is emitted. You can specify logic in the callback to process
-      the change event document when it is received.
+      The listener function triggers when a change event is emitted. You
+      can specify logic in the listener to process the change event document
```

Comment:
Issue: [Hyphenate compound modifiers](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#hyphens-in-compound-modifiers) (assuming this is a "document" of the type "change event", rather than an "event document" of the type "change").

```suggestion
      can specify logic in the listener to process the change-event document
```

Reply:
It's a `change` event.